### PR TITLE
Configurations for NuMI Analysis

### PR DESCRIPTION
**Pull request by gputnam**
Configurations to enable NuMI analysis. Detector variations and systematic reweights.

Update: I've removed the geant4reweight code. This is now a standalone PR.

### CODE REVIEW

**Comment by SFBayLaser** [comment]
The wirecell files needed for this PR are now in icarus_data tagged as v09_83_01

**Comment by mmrosenberg** [comment]
trigger build SBNSoftware/sbncode#417

File: icaruscode/TPC/ICARUSWireCell/detsimmodules_wirecell_ICARUS.fcl [hunk 1]
```diff
@@ -80,4 +81,16 @@ icarus_simwire_wirecell_fitSR.wcls_main.structs.gain0: 11.9918701 # mV/fC
...
+icarus_simwire_wirecell_fitSR_P0nom: @local::icarus_simwire_wirecell
+# Add in the ER tail
+icarus_simwire_wirecell_fitSR_P0nom.wcls_main.params.file_rcresp: "icarus_fnal_rc_tail.json"
+# Add in the tuned field responses (except for P0)
+icarus_simwire_wirecell_fitSR_P0nom.wcls_main.params.files_fields: "icarus_fnal_fit_ks_P0nom.json.bz2"
+# futz with shaping+gain values (note these are really just scale factors and should not be taken literally)
+icarus_simwire_wirecell_fitSR_P0nom.wcls_main.structs.gain0: 17.05212 # mV/fC 
+icarus_simwire_wirecell_fitSR_P0nom.wcls_main.structs.gain1: 12.6181926 # mV/fC
+icarus_simwire_wirecell_fitSR_P0nom.wcls_main.structs.gain2: 13.0261362 # mV/fC
+icarus_simwire_wirecell_fitSR_P0nom.wcls_main.structs.shaping1: 1.45 # us
```
Comment (SFBayLaser) [inline icaruscode/TPC/ICARUSWireCell/detsimmodules_wirecell_ICARUS.fcl:95]: H Gray, while all the updates in this PR look to be specific to NuMI analysis... won't this one affect all detector simulations for ICARUS? Andif this is the correct updates for this file shouldn't it have its own PR with an explanation so we know when/where and why it is happening?

**Comment by mmrosenberg** [comment]
trigger build LArSoft/lar*@LARSOFT_SUITE_v09_90_00

**Pull request merged**